doc/classes/DirAccess.xml

```diff
@@ -253,7 +253,35 @@
 			<param index="0" name="path_a" type="String" />
 			<param index="1" name="path_b" type="String" />
 			<description>
-				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object. Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (e.g., identical copies of the file that are not symbolic links).
+				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object (taking into account case sensitivity, relative paths, symbolic links, and hard links). Returns [code]false[/code] otherwise, even if the files are bit-for-bit identical (i.e. identical copies of a file that are not symbolic links).
```

Comment:
One time it's `i.e.`, another time it's `e.g.`. I think we should begin not using both outright.
```suggestion
				Returns [code]true[/code] if paths [param path_a] and [param path_b] resolve to the same file system object (taking into account case sensitivity, relative paths, symbolic links, and hard links). Returns [code]false[/code] otherwise, even if the files are exactly identical (copies of a file that are not symbolic links).
```